Pipeline/CreateUCDComponentVersion/README.md

```diff
@@ -1,69 +1,237 @@
-# IBM UCD Packaging based on IBM DBB Build Report
+# IBM UCD Packaging Based on IBM DBB Build Report
 
-An important step in the pipeline is to generate a deployable package. This sample groovy script
-- extracts information about the build outputs from the DBB BuildReport.json
-- generates the UCD shiplist ```shiplist.xml``` file
-- invokes the ```buztool.sh``` of UCD
+## Summary
+
+An important step in the pipeline is to generate a deployable package. This sample Groovy script:
+
+- Extracts information about the build outputs from the Dependency Based Build (DBB) `BuildReport.json`. The script is able to take a single DBB build report or multiple build reports to build a cumulative package across multiple incremental builds. 
+- Generates the UrbanCode Deploy (UCD) shiplist `shiplist.xml` file.
+- Invokes the `buztool.sh` with the appropriate configuration to store the binary package in the artifact repository and to register a new UCD component version.
+
+## High-level Processing Flow
+
+1. **Initialization**
+   1. Read command line parameters.
+   1. (Optional) Read any application and global properties that are passed via `--packagingPropFiles`.
```

Comment:
I revised this line to try and make it a bit more concise and clear. Please feel free to edit it if you feel the information has lost any accuracy.

The original was:
> Read application and global properties which are supposed to be passed via `--packagingPropFiles` (Optionally)